macos/README.md

```diff
@@ -13,30 +13,30 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 ## Tools needed to build the package
 
 To build a Wazuh package you need to install the following tools:
-  - [Packages](http://s.sudre.free.fr/Software/Packages/about.html): You can install this on OSX using the generate_wazuh_packages.sh script in this repo.
-  - `git`: on OSX install with homebrew use `brew install git`
+- [Packages](http://s.sudre.free.fr/Software/Packages/about.html): You can install this on OSX using the generate_wazuh_packages.sh script in this repo.
```

Comment:
Please change OSX to macOS. All references.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/162/commits/4c4b8cfd1a572c0993e15c0f7e953f2f249a79a0